packages/odie-client/src/hooks/use-create-zendesk-conversation.ts

```diff
@@ -13,11 +13,13 @@ export const useCreateZendeskConversation = (): ( ( {
 	interactionId,
 	section,
 	createdFrom,
+	fromError,
```

Comment:
Another suggestion to be consistent with how we usually name _boolean_ variables:
```suggestion
	isFromError,
```